app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/capture_context.rb

```diff
@@ -8,7 +8,7 @@ def initialize(target, start_time, end_time, interval)
       @end_time = end_time
       @interval = interval
       @tenant = target.try(:container_project).try(:name) || '_system'
-      @ext_management_system = @target.ext_management_system
+      @ext_management_system = @target.ext_management_system || @target.try(:old_ext_management_system)
```

Comment:
>I have the feeling that you need to change also this part of the code here:
>
>  @ext_management_system = @target.ext_management_system
Because then gets validated here
>
>def validate_target
>  raise TargetValidationError, "Validation error: ems not defined"    unless @ext_management_system
>  ...
>end

@simon3z